rmgpy/molecule/atomtype.py

```diff
@@ -548,7 +548,7 @@ def getFeatures(self):
 
 atomTypes['F'  ] = AtomType('F',   generic=['R','R!H','Val7'],      specific=['F1s'])
 atomTypes['F1s'] = AtomType('F1s', generic=['R','R!H','F','Val7'],  specific=[],
-                             single=[0,1], allDouble=[0], rDouble=[], oDouble=[], sDouble=[], triple=[0], benzene=[0], lonePairs=[3], charge=[0])
+                             single=[0,1], allDouble=[0], rDouble=[], oDouble=[], sDouble=[], triple=[0], quadruple=[0], benzene=[0], lonePairs=[3], charge=[0])
```

Comment:
The default value for `quadruple` is an empty list. Shouldn't that be enough (I guess not from your experience with F, but I wonder why)?
Perhaps a quicker fix for all elements would be to change the default behaviour of AtomType.
I suggest editing [line 118](https://github.com/ReactionMechanismGenerator/RMG-Py/blob/master/rmgpy/molecule/atomtype.py#L118) to:
`self.quadruple = quadruple or [0]`
so zero quadruple bonds becomes the default.

Reply:
Actually, we don't know that the default was causing a problem, we just thought we'd make Fluorine the same as Chlorine. I think our actual problem was solved by the other change.